Doc/library/stdtypes.rst

```diff
@@ -1796,25 +1864,60 @@ expression support in the :mod:`re` module).
    Return ``True`` if all characters in the string are alphanumeric and there is at
    least one character, ``False`` otherwise.  A character ``c`` is alphanumeric if one
    of the following returns ``True``: ``c.isalpha()``, ``c.isdecimal()``,
-   ``c.isdigit()``, or ``c.isnumeric()``.
+   ``c.isdigit()``, or ``c.isnumeric()``::
+
+      >>> ''.isalnum()
+      False
+      >>> 'abc123'.isalnum()
+      True
+      >>> 'abc123!@#'.isalnum()
+      False
+      >>> ' '.isalnum()
+      False
+
 
 
 .. method:: str.isalpha()
 
    Return ``True`` if all characters in the string are alphabetic and there is at least
    one character, ``False`` otherwise.  Alphabetic characters are those characters defined
    in the Unicode character database as "Letter", i.e., those with general category
-   property being one of "Lm", "Lt", "Lu", "Ll", or "Lo".  Note that this is different
+   property being one of "Lm", "Lt", "Lu", "Ll", or "Lo"::
+
+      >>> 'a commom word'.isalpha()
+      False
+      >>> 'acommomword'.isalpha()
+      True
+      >>> 'µ'.isalpha()
+      True
+      >>> 'æ'.isalpha()
+      True
+      >>> 'Ŧ'.isalpha()
+      True
+
+   Note that this is different
    from the `Alphabetic property defined in the section 4.10 'Letters, Alphabetic, and
    Ideographic' of the Unicode Standard
-   <https://www.unicode.org/versions/Unicode15.1.0/ch04.pdf>`_.
-
+   <https://www.unicode.org/versions/Unicode15.1.0/ch04.pdf>`_
+   See Unicode Properties section in :ref:`unicode-howto`.
```

Comment:
We could add a reference label to that section and then link to it directly.

https://devguide.python.org/documentation/markup/#doc-ref-role

Reply:
good suggestion, thank you!